demo/src/javascripts/components/demos/arc.js

```diff
@@ -108,16 +111,14 @@ export default class ArcDemo extends Component {
     selectedFeature = selectedFeature || features[362];
 
     const {flows, centroid, name} = selectedFeature.properties;
-    const arcs = [];
-
-    for (let toId in flows) {
+    const arcs = Object.keys(flows).reduce((acc, toId) => {
       const f = features[toId];
-      arcs.push({
+      return [...acc, {
```

Comment:
While elegant, I think there could be a big performance difference here as the new code creates a new array every time. I would consider restoring the old code.